### PR TITLE
Noesis/UnityMVVM against movra/UnityMVVM

### PR DESCRIPTION
**Pull request by movrajr**

### CODE REVIEW

**Comment by jesusdesantos** [comment]
Now it seems to be fine! Comments:
- .meta files cannot be deleted. We use them to associate a GUID that is going to be shared with uFrame. That way, both packages can have the folder MVVM whenever they want without generating conflict between .unitypackages
-  notifycollectionchangedeventargs.cs should be renamed to NotifyCollectionChangedEventArgs.cs

**Comment by movrajr** [comment]
- OK, will re-add the .meta files.
- The Microsoft Reference Source has inconsistent capitalization in its filenames. 

https://github.com/Microsoft/referencesource/blob/9da503f9ef21e8d1f2905c78d4e3e5cbb3d6f85a/System/compmod/system/collections/specialized/inotifycollectionchanged.cs

https://github.com/Microsoft/referencesource/blob/9da503f9ef21e8d1f2905c78d4e3e5cbb3d6f85a/System/sys/system/windows/input/ICommand.cs

**Comment by movrajr** [comment]
Done.

**Comment by jesusdesantos** [comment]
We need the same GUIDs we assigned. You changed them :)

**Comment by jesusdesantos** [comment]
Regarding the capitalization of notifycollectionchangedeventargs I think we should change it because it doesn't follow our coding conventions. It doesn't matter if Microsoft was not able to be coherent.

**Comment by movrajr** [comment]
Now it's a big mess. Give me a minute.

**Comment by movrajr** [comment]
Now they should have the same GUIDs as in this uFrame pull request https://github.com/InvertGames/uFrame/pull/23

**Comment by jesusdesantos** [comment]
They still are different from the one we have in our package. For example, ICommand.cs.meta must be:

guid: 286c9c0f4dda27b4f8940957677e6ec0

File: ICommand.cs.meta [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 286c9c0f4dda27b4f8940957677e6ec0
```
Comment (jesusdesantos) [inline ICommand.cs.meta:2]: This guid cannot be changed

File: INotifyCollectionChanged.cs.meta [hunk 2]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 12a97e7948c8d4342925e0635dcac469
```
Comment (jesusdesantos) [inline INotifyCollectionChanged.cs.meta:2]: This guid cannot be changed

File: ObservableCollection.cs.meta [hunk 3]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 65b19a9acc0ecbd4c9adcfd6167240a0
```
Comment (jesusdesantos) [inline ObservableCollection.cs.meta:2]: This guid cannot be changed

**Pull request merged**